pkg/webhookconfig/registration.go

```diff
@@ -167,7 +167,7 @@ func (wrc *Register) Remove(cleanUp chan<- struct{}) {
 	}
 
 	wrc.removeWebhookConfigurations()
-	wrc.removeSecrets()
+	// wrc.removeSecrets()
```

Comment:
Upon termination, we need to clean up these secrets. Do we need this?

Reply:
We might need to. Here's a scenario:
1. Pod 1 (Leader) dies, deletes the secret.
2. Pod 2 (Assumes leadership) and continues to work with the old certificates.
3. Pod 1 is restarting and sees the Secret is not present, so it creates one.

This way, we have inconsistent certificates.

If we don't clean up on pod deletion, in the future we can cross-check the annotation to determine if the Secret is an old one and we need to create another.

Reply:
With this condition, managed resources are deleted only when Kyverno is terminating:
```
	if !wrc.cleanupKyvernoResource() {
		return
	}
```

Will the above scenario occur?